tests/ballerina-unit-test/src/test/resources/test-src/expressions/async/basic-async-operations.bal

```diff
@@ -79,8 +79,17 @@ function testAsyncNonNativeBasic9() returns int {
 
 function testAsyncNonNativeBasic10() returns any {
   future<int> f1 = start addNum(5, 2);
-  future<int> f2 = start addNum(5, 2);
+  future<int> f2 = start subtract(5, 2);
+  boolean cancelled_f2 = f2.cancel();
+  any result = wait {f1,f2};
+  return result;
+}
+
+function testAsyncNonNativeBasic11() returns any {
+  future<int> f1 = start addNum(5, 2);
```

Comment:
I think you need to use subtract method here as well, (you cannot guarantee that addNum method will not be finished before calling cancel right?)

Reply:
yeah +1 using the subtract method ensures that the method will never be finished. Added it with 738541d. Thank you :)